CExts.c

```diff
@@ -0,0 +1,59 @@
+//
+//  CExts.cpp
```

Comment:
Has this file been taken from another open source project? If so, I'd add a reference to it.

Reply:
no, this is my own